packages/ember-glimmer/tests/integration/outlet-test.js

```diff
@@ -278,4 +278,50 @@ moduleFor('outlet view', class extends RenderingTest {
 
     this.assertText('HIBAR');
   }
+
+  ['@test outletState can pass through user code (liquid-fire initimate API) ']() {
+    this.registerTemplate('outer', 'A{{#-with-dynamic-vars outletState=(identity (-get-dynamic-var "outletState"))}}B{{outlet}}D{{/-with-dynamic-vars}}E');
+    this.registerTemplate('inner', 'C');
+
+    // This looks like it doesn't do anything, but its presence
+    // guarantees that the outletState gets converted from a reference
+    // to a value and then back to a reference. That is what we're
+    // testing here.
+    this.registerHelper('identity', ([a]) => a);
```

Comment:
Can you make this do _something_ instead of pass-through? This allows you to test updating behavior (see below)